views/installation/page/elements/sidebar.php

```diff
@@ -29,7 +29,7 @@
 echo '</ol>';
 
 $options_values = [];
-foreach (get_installed_translations() as $key => $value) {
+foreach (elgg()->translator->getInstalledTranslations() as $key => $value) {
```

Comment:
I believe it's more efficient to first store the result in a var and loop through that. don't know for sure though

Reply:
i think not